engine/tests/mechanics/games/test_credentials_factory.py

```
@@ -0,0 +1,112 @@
+    Restrictions,
+    RestrictionLevel,
+    applicable_modes,
+    build_valid,
+    degrade,
+    derive_disposition,
+    make_case,
+    sample_failure_mode,
+)
+
+D = CredentialDisposition
+I = Indication
```

Ruff (0.15.13)

[error] Ambiguous variable name: `I`

(E741)